### DISCUSSION TITLE
Completely remove organization

### DISCUSSION THREAD

**Discussion by corneleberle-pro** [zitadel/zitadel#6348] [Q&A]
When I remove an organization it stays in the list with state "Removed". Is there a way to completely get rid of this organization or at least to hide it from the organizations list overview and the organizations dropdown?

**Comment by fforootd** [comment]
There is current work to delete organizations located here https://github.com/zitadel/zitadel/pull/6083

So we are working on this but the ETA is yet unclear.

**Reply by corneleberle-pro** [reply]
I see. thanks for clarifying.
